common.go

```diff
@@ -42,11 +42,16 @@ type JobIter interface {
 
 // RepositoryID tries to find a repository by the endpoint into the database.
 // If no repository is found, it creates a new one and returns the ID.
-func RepositoryID(endpoint string, storer *model.RepositoryStore) (uuid.UUID, error) {
+func RepositoryID(endpoints []string, storer *model.RepositoryStore) (uuid.UUID, error) {
```

Comment:
Just wondering if `GetOrCreateRerpositoryID` would possibly describe the purpose of this function any better